packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -63,12 +63,44 @@ class Autocomplete<T extends Object> extends StatelessWidget {
     required this.optionsBuilder,
     this.displayStringForOption = RawAutocomplete.defaultStringForOption,
     this.fieldViewBuilder = _defaultFieldViewBuilder,
+    this.focusNode,
     this.onSelected,
+    this.textEditingController,
     this.optionsMaxHeight = 200.0,
     this.optionsViewBuilder,
     this.optionsViewOpenDirection = OptionsViewOpenDirection.down,
     this.initialValue,
-  });
+  }) : assert((focusNode == null) == (textEditingController == null)),
+       assert(
+         !(textEditingController != null && initialValue != null),
+         'textEditingController and initialValue cannot be simultaneously defined.',
+       );
+
+  /// The [FocusNode] that is used for the text field.
+  ///
+  /// {@template flutter.widgets.RawAutocomplete.split}
```

Comment:
This template syntax allows you to reuse these docs as a macro. It's already declared in RawAutocomplete, so you shouldn't redeclare it here.

I think what you should do is adjust it in RawAutocomplete so it doesn't include the example, then use it here as a macro (`/// @macro flutter.widgets.RawAutocomplete.split`), and then point to RawAutocomplete.focusNode in a "See also".